core/primitives/src/version.rs

```diff
@@ -52,6 +52,9 @@ pub const DELETE_KEY_STORAGE_USAGE_PROTOCOL_VERSION: ProtocolVersion = 40;
 
 pub const SHARD_CHUNK_HEADER_UPGRADE_VERSION: ProtocolVersion = 41;
 
+/// Updates the way receipt ID is constructed to use current block hash instead of last block hash
+pub const CREATE_RECEIPT_ID_SWITCH_TO_CURRENT_BLOCK_VERSION: ProtocolVersion = 48;
```

Comment:
@evgenykuzyakov do we want to go through nightly protocol for fixes like this as well?

Reply:
I don't think so. It seems simple enough and covered by unit tests